data-protocols/ids/ids-core/src/main/java/org/eclipse/dataspaceconnector/ids/core/IdsCoreServiceExtension.java

```diff
@@ -60,7 +58,7 @@ public void initialize(ServiceExtensionContext context) {
         context.registerService(IdsDescriptorService.class, descriptorService);
 
         var identityService = context.getService(IdentityService.class);
-        var connectorName = getConnectorId(context);
+        var connectorName = context.getConnectorId();
```

Comment:
better call var connectorId (there are a few more places where connectorName is used...)

Reply:
OK updated